### ISSUE TITLE
JPEGDecompressSurface : Picture decode failed: e00002d1

### ISSUE THREAD

**Issue by domanc** [rs/SDWebImage#2109]
### New Issue Checklist

JPEGDecompressSurface : Picture decode failed: e00002d1
### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / tvos
 Platform Version        | e.g. 8.0
 SDWebImage Version      | e.g. 4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 9
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by domanc** [comment]
Handed the images back to the server URL with sdwebimage loading is showed  JPEGDecompressSurface : Picture decode failed: e00002d1 

When you use the web page to load the url is a data flow, and then use sdwebimage decoding failure load

**Comment by dreampiggy** [comment]
Don't know the reason. It's a error log from Apple's close-source JPEG decoder.(Not libjpeg). Could you please provide that image or view that with third-party image viewer ?
